blog/from-maid-to-development.us.md

```diff
@@ -0,0 +1,24 @@
+---
+author: "4GeeksAcademy"
+date: "2019-03-20T05:29:14+00:00"
+excerpt: "Intrigued by the prospect of launching a  rewarding career without a college degree, Valerie explored her options and found 4Geeks Academy to be the perfect fit."
+template: "post" 
+title: "From Maid to Developer: Valerie's Stellar Journey at 4Geeks Academy"
```

Comment:
Cambiar titulo a: "From Cleaning Houses to Developer : Valerie's Stellar Journey at 4Geeks Academy"